guides/common/modules/proc_installing-the-iop-advisor-engine-in-a-disconnected-environment.adoc

```diff
@@ -0,0 +1,27 @@
+[id="installing-the-iop-advisor-engine-in-a-disconnected-environment"]
+= Installing the IoP Advisor Engine in a disconnected environment
+
+.Prerequisites
+* Ensure you have the {Project} installation ISO downloaded or obtain the exported container image.
```

Comment:
I'm not sure you need to mention this as a prerequisite when the user has already downloaded the binary images in 2.1.